packages/syft/src/syft/core/node/new/user_service.py

```diff
@@ -138,6 +147,39 @@ def update(
         if user is None:
             return SyftError(message=f"No user exists for given UID: {uid}")
 
+        if (
```

Comment:
I think we can remove explicit check for DATA_OWNER, by adding a check for ADMIN.

```
if user.role == ServiceRole.ADMIN:
   #DO ANYTHING
elif user_update and ((context.role <= user.role) or (context.role <= user_update.role)):
	   # Cannot update if:
		   # role lower than equal to your role 
		   # or your role is lower than role to be updated

```